packages/deployer/src/server/index.ts

```diff
@@ -2118,16 +2118,16 @@ export async function createHonoServer(
     deleteIndex,
   );
 
-  app.get(
-    '/openapi.json',
-    openAPISpecs(app, {
-      documentation: {
-        info: { title: 'Mastra API', version: '1.0.0', description: 'Mastra API' },
-      },
-    }),
-  );
-
-  app.get('/swagger-ui', swaggerUI({ url: '/openapi.json' }));
```

Comment:
This line was preventing the already existing `options.swaggerUI` from effectively turning off the swagger endpoints